Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/Dockerfile-torchserve

```diff
@@ -22,7 +22,8 @@ RUN mkdir -p ${MLFLOW_HOME}/scripts && \
     git clone https://github.com/pytorch/serve.git && \
     cd serve && python ./ts_scripts/install_dependencies.py && \
     pip install --no-cache-dir torch==1.7.1+cpu torchvision==0.8.2+cpu torchaudio==0.7.2 torchserve torch-model-archiver -f https://download.pytorch.org/whl/torch_stable.html && \
-    pip install --no-cache-dir -r /tmp/requirements.txt
+    pip install --no-cache-dir -r /tmp/requirements.txt \
```

Comment:
Curious how this works taking into account that:

1. There's no `requirements.txt` in the container as far as I understand. Only `requirements-torchserve.txt`. Or is there?
2. More importantly, there's no `&&` at the end of the line, so the resulting command is `... pip install --no-cache-dir -r /tmp/requirements.txt pip install --no-cache-dir -r /tmp/requirements-torchserve.txt`

Perhaps I should just try the branch locally (which I will do a bit later)

Reply:
1. The two files get copied in
```COPY requirements-torchserve.txt requirements.txt /tmp/```
2. Hmm, that is weird...

Reply:
Ah, I misread the `COPY` line, thanks for pointing it out.

Reply:
https://stackoverflow.com/questions/36744143/how-can-i-define-multiple-requirement-files
Looks like it will install all packages in the two files [plus pip plus a package called install (-_-)]

Reply:
```Installing collected packages: pycparser, pyasn1, oauthlib, MarkupSafe, Werkzeug, smmap, rsa, requests-oauthlib, pyasn1-modules, protobuf, Jinja2, itsdangerous, isodate, click, cffi, cachetools, websocket-client, tabulate, sqlalchemy, pytz, python-editor, prometheus-client, msrest, Mako, googleapis-common-protos, google-auth, gitdb, Flask, cryptography, azure-core, threadpoolctl, sqlparse, scipy, regex, querystring-parser, pyyaml, prometheus-flask-exporter, pandas, joblib, gunicorn, google-crc32c, google-api-core, gitpython, entrypoints, docker, databricks-cli, cloudpickle, azure-storage-blob, alembic, tokenizers, scikit-learn, sacremoses, mlflow, google-resumable-media, google-cloud-core, filelock, transformers, sklearn, mlflow-torchserve, install, google-cloud-storage```